src/wiki/forms_account_handling.py

```diff
@@ -0,0 +1,89 @@
+import random
+import string
+
+import django.contrib.auth.models
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import FieldDoesNotExist
+from django.db.models.fields import CharField, EmailField
+from django.utils.translation import gettext_lazy as _
+from wiki.conf import settings
+
+
+def _get_field(model, field):
+    try:
+        return model._meta.get_field(field)
+    except FieldDoesNotExist:
+        return
+
+
+User = get_user_model()
+
+
+def check_user_field(user_model=User):
+    return isinstance(_get_field(user_model, user_model.USERNAME_FIELD), CharField)
+
+
+def check_email_field(user_model=User):
```

Comment:
`User` is assigned as a default kwarg, but it seems to always be called w/o kwargs?

Reply:
Its always called with the result of get_user_model() ( either with default kwarg, or in [wiki/checks.py](https://github.com/django-wiki/django-wiki/blob/68f7dc9b14478eb0370438d23de40c7e64521642/src/wiki/checks.py) also ).

Having the parameter makes it re-usable, it could check a custom user model is valid also. I'd probably prefer to make sure all calls always pass user_model, and remove the default kwargs. I'll look into doing that.